gui/doc/slider.csv

```diff
@@ -17,10 +17,15 @@ Possible values are:
 labels,bool|dict,,"The labels for specific points of the slider.
 If set to <i>True</i>, this slider uses the labels of the <i>lov</i> if there are any.
 If set to a dictionary, the slider uses the dictionary keys as a <i>lov</i> key or index, and the associated value as the label."
+continuous,bool,True,"If set to <i>False</i>, the control emits an on_change notification only when the cursor is released, 
+otherwise notifications are emitted during the cursor movements. 
+If <i>lov</i> is defined, the default value is <i>False</i>."
+change_delay,int,<i>App config</i>,"minimum time between triggering two <i>on_change</i> callback.
+The default value is defined at the application configuration level by change_delay. if None or 0, there's no delay."
```

Comment:
"configuration level by change_delay"
->
<pre>
configuration level by the **change_delay** configuration option
</pre>